manifests/system-podsecuritypolicy.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: default-psp
```

Comment:
is someone going to complain about the 'default-psp' being so permissive? Would it be worth naming it something like 'system-permissive-psp' so that the user can provide their own default without getting confused?

Reply:
This is to get it in place and provide the ability for RKE2 to even deploy itself. The restrictiveness is on the customer to decide based on their needs, controls, etc.

Reply:
Yeah, mostly just thinking about it from a perception perspective. Explaining to an auditor why the default is so privileged, vs something with a name that makes it clear that it's used by core system components.

Reply:
This is going to be updated to comply with CIS reqs in another PR soon. However, even if it were left open, the onus of responsibility is on the customer per their compliance requirements which will vary by a number of factors.